docs/dev/testing.md

```diff
@@ -0,0 +1,38 @@
+# Testing Pull Requests 
+
+One way to help pyDARN get developed faster and smoothly is by testing [Pull Requests](https://github.com/SuperDARN/pydarn/pulls). 
+Sometimes this can take fifteen minutes to do which helps the developer merge their code in 
+and allowing you on the author list. 
+
+Every contribution helps! 
+
+## Setting up the testing environment
+
+1. clone the [pyDARN repository](https://github.com/SuperDARN/pydarn.git)
+  `git clone https://github.com/SuperDARN/pydarn.git`
+2. Then checkout the branch you need to test or test against. The author of the Pull Request should give you some information on how to test the code and if you need to checkout other branches to compare results. To checkout a branch you want to test use this git commands:
+    ```bash
+    git fetch 
+    git checkout <branch name>
+    git pull origin <branch name>
+    ```
+
+    !!! Note 
+        if you are unaware what the branch name is at the top of the page below the Pull Request title and number shows the branch name and which branch it will be merged into. 
+
+3. From here you just need to install the code run the normal installation steps in a python environment (this ensures it doesn't mix with your current pyDARN version)
```

Comment:
Add a link to a how to python venv? I don't really know a good one, but I usually go to the pyDARN docs when I forget :P